en/drafts/originals/designing-a-timeline-tabletop-simulator.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: "Designing a _Timeline_ for the Tabletop and Tabletop Simulator"
+title: "Designing a _Timeline_ card deck for tabletops and Tabletop Simulator"
```

Comment:
Hi @copystar and @RolRodr, I've suggested a slight alteration to the lesson title. Does this make good sense to you both?
I'm also not sure we can use italics in the lesson title for _Timeline_ – we'll see what it looks like in the preview, once I've merged this branch.

Reply:
I'm okay with this title change!

Reply:
Another alternative is "Designing a deck of _Timeline_ cards for tabletops and Tabletop Simulator"

Reply:
I like this second suggestion better!  I like that it carries the original 'tabletops and tabletop simulator'